.github/workflows/ci.yaml

```diff
@@ -29,25 +29,28 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04]
+        os: [ubuntu-22.04, ubuntu-24.04]
         build-type: [Debug, RelWithDebInfo]
-        standalone: [standalone, catkin]
-        solvers-from-source: [SolversFromSource, SolversFromAPT]
```

Comment:
I would appreciate it if you could add a comment on why you deleted SolversFromAPT and how it can be added back.